lib/rules/eol-last.js

```diff
@@ -5,9 +5,9 @@ import Rule from './_base.js';
 
 export default class EolLast extends Rule {
   parseConfig(config) {
-    // In strict mode (= template is embedded in e.g. a JS file) we want to disable this rule,
```

Comment:
strict mode is not the same as embedded! `hbs` tagged template is embedded, but not strict!

As strictMode was fixed here, which now behaves correctly but differently, I needed to pass `isEmbedded` to rules to get this right here!